### ISSUE TITLE
tenant onboarding failed with create_complete in cloudformation

### ISSUE THREAD

**Issue by qwudssd** [awslabs/aws-saas-boost#215]
### The Question
Tenant onboarding process failed shortly after created with no error appearing in CloudFormation stacks. But with some errors showing up in cloudwatch logs, not sure if they are related to the failure of the tenant onboarding.
<img width="1085" alt="Screen Shot 2022-05-04 at 12 12 19 PM" src="https://user-images.githubusercontent.com/104920467/166725335-348347a0-e46c-44b5-adea-7ac83f8ddc37.png">

### Environment

  - **AWS SaaS Boost Version:** v2.0.0-rc1

**Comment by PoeppingT** [comment]
Hi @qwudssd , thanks for opening this issue. To help we'll need some more information, as your attached log lines are a red herring in the Onboarding Service logs (although if you can reliably reproduce I suggest you open a bug with exact steps to show how this happened). 
1. Have you made any changes to your local SaaS Boost installation?
2. What tenant CloudFormation stacks do you have? There should be at least two: a `sb-<env>-tenant-<id>` and a `sb-<env>-tenant-<id>-app-<serviceName>-<serviceId>`
3. Try taking a look at the logs for the `sb-<env>-onboarding-events` lambda. This is the lambda that handles the Onboarding workflow once the initial stack has been created and should create the secondary stack.

**Comment by qwudssd** [comment]
Thanks for the response! 
1. I haven't made any changes to the local saas boost installation
2. And looks like there is only one tenant cloudformation stack: ```sb-<env>-tenant-<id>```, ```sb-<env>-tenant-<id>-app-<serviceName>-<serviceId>``` was not being created
3. So in the logs for the ```sb-<env>-onboarding-events``` lambda, besides that nullpointer exception there is another error
<img width="1116" alt="Screen Shot 2022-05-05 at 10 40 18 AM" src="https://user-images.githubusercontent.com/104920467/166948368-32957b51-2659-49eb-a1ff-3f10aa251828.png">

**Comment by PoeppingT** [comment]
Thanks for the clarification, this bug is actually already fixed in the multi-container branch and will be fixed in the rest release.

To unblock yourself right away, try either updating or reinstalling your environment using the multi-container branch.

**Comment by brtrvn** [comment]
Merged into `main`
